packages/core/protocols/src/proto/dxos/client/services.proto

```diff
@@ -442,6 +442,12 @@ message Invitation {
 
   /// Path or identifier to navigate to after successful authentication.
   optional string target = 11;
+
+  /// Host should resume invitation on startup until timeout.
+  optional bool persistent = 12;
+
+  /// Host should resume invitation on startup until expiry.
+  optional google.protobuf.Timestamp persistence_expiry = 13;
```

Comment:
1. consider adding a timestamp field for the creation and a separate expiration or TTL field. also the field could just be `expiration`. e.g., we might want a policy to set a max TTL, so we'd need the creation timestamp for that.

2. is it required to have a field to designated "persistent" i.e., is that usefully part of the data structure. from the usage below it could be an option on the method. IOW, what is the use case: does the user know ahead of time that they are creating a persistent invitation?

3. how is that different from TYPE = OFFLINE? in the OFFLINE docs above we required a public key -- is there a different auth method we can implement? e.g., requiring a PIN. @dmaretskyi?